data_generators/user_generators.py

```diff
@@ -14,10 +15,12 @@ def __str__(self):
 
 
 class RandomMentor(Printable):
-    """Generates Mentor record"""
+    """Generates a mock Mentor record"""
 
     def __init__(self):
         self.profile_id = generate_uuid(16)
+        self.created_at = datetime.now().isoformat()
```

Comment:
Having mock data timestamped at the moment of generation might not be a good thing. We need data that is spread out over time and not all on top of each other.

Reply:
@sspradling78 Can you please provide clarity on what we will need spread out timestamps for? As well, please review the mentee update with this requested change. Or is mentee specifically fine with a static timestamp? I think we'll need to sync up on this as an entire team rather than delay this PR.

Reply:
Ya, let's discuss what we could do. If we timestamp all mentors and mentees the same timestamp then we will have a problem with the graphs our team is trying to implement.

Reply:
@sspradling78 great thanks for the clarification. Can you please remove this change until we get a unified decision? It appears mentee generator update doesn't have the same requested change.

Reply:
There's still one more requested change.